website/static/urlValidatorTest.json

```diff
@@ -32,12 +32,12 @@
     "http://مثال.إختبار":"should accept valid unicode heavy website test 1",
     "http://例子.测试":"should accept valid unicode heavy website test 2",
     "http://उदाहरण.परीक्षा":"should accept valid unicode heavy website test 3",
-    "http://-.~_!$&()*+,;=:%40:80%2f::::::@example.com":"should accept valid website with user but crazy username",
+    "http://-.~_!%24%26()*%2B%2C%3B%3D%3A%40%3A80%2F%3A%3A%3A%3A%3A%3A@example.com":"should accept valid website with user but crazy username",
     "http://1337.net":"should accept valid website with just numbers in domain",
     "http://definitelyawebsite.com?real=yes&page=definitely":"should accept valid website with query",
     "http://a.b-c.de":"should accept valid website with dash",
     "http://website.com:3000/?q=400":"should accept valid website with port and query",
-    "http://asd/asd@asd.com/":"should accept valid website with unusual username"
+    "http://asd%2Fasd@asd.com/":"should accept valid website with unusual username"
```

Comment:
Django 3 is more strict on special characters encoding in URL and we should follow the spec.